device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -211,8 +251,34 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin {
             ekEvent!.startDate = startDate
             ekEvent!.endDate = endDate
             ekEvent!.calendar = ekCalendar!
+            
+            let recurrenceRuleArguments = arguments[recurrenceRuleArgument] as? Dictionary<String, AnyObject>
```

Comment:
Should we first check if the key is present in the dict? Wonder if this can cause exceptions?

Reply:
no it wont throw. this is how it's done in Swift and Objective-C as if the key's not there then it'll return nil